lib/tasks/data.rake

```diff
@@ -0,0 +1,69 @@
+DATA_MIGRATION_SERVICES = [
+  # do not delete or edit this line - services added below by generator
+].freeze
+
+def data_migrations
+  puts 'Running data migrations configured in lib/tasks/data.rake...'
+
+  DATA_MIGRATION_SERVICES.each do |data_migration_service_name|
+    service = Object.const_get(data_migration_service_name)
+
+    next if service::MANUAL_RUN || migration_ran?(service)
+
+    puts "Executing #{service}..."
+    ActiveRecord::Base.transaction do
+      log_migration(service)
+      service.new.change
+    end
+    puts "#{service}##{service::TIMESTAMP} data migration completed successfully"
+  end
+end
+
+def migration_ran?(service)
+  DataMigration.exists?(service_name: service.to_s, timestamp: service::TIMESTAMP)
+end
```

Comment:
Could we possibly pull these up into the `DataMigration` class or some other namespace so we can test them?

Reply:
Added a DataMigrationRunner service and tests to verify the functionality.